cmd/root.go

```diff
@@ -49,9 +57,12 @@ func init() {
 	cobra.OnInitialize(initConfig)
 
 	// Placeholder for config file
-	//rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.lhctl.yaml)")
+	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.lhctl.yaml)")
 	// root persistent flag `url` will be available everywhere in the package
-	rootCmd.PersistentFlags().StringVar(&url, "url", "", "longhorn manager url (example: http://10.88.1.3/v1)")
+	rootCmd.PersistentFlags().StringVar(&urlFlag, "url", "", "longhorn manager url (example: http://10.88.1.3/v1)")
+	rootCmd.PersistentFlags().StringVar(&userFlag, "user", "", "user for http request")
+	rootCmd.PersistentFlags().StringVar(&passFlag, "pass", "", "password for http request")
+	rootCmd.PersistentFlags().StringVar(&contextFlag, "context", "", "config file context")
 	//rootCmd.MarkFlagRequired("url")
```

Comment:
IMO, unused code should be removed before reaching master - not just commented out.